pkgs/top-level/all-packages.nix

```diff
@@ -497,6 +499,8 @@ with pkgs;
 
   frugal = callPackage ../development/tools/frugal { };
 
+  gclient-wrapped = callPackage ../development/tools/gclient-wrapped {};
```

Comment:
rename + move to `gclient`

Reply:
It's a wrapper, wouldn't it make sense to called it `gclient-wrapped` like all of the other wrapped things in nixpkgs?

Reply:
all binaries are wrapped in nixpkgs

its an exception when they are UNwrapped, like clang-unwrapped

Reply:
That kinda makes sense except `kubernetes-helm-wrapped` is one that has wrapped in its name.